tabbycat/options/preferences.py

```diff
@@ -1059,6 +1059,27 @@ class SplitVotingBallots(BooleanPreference):
 # ==============================================================================
 
 
+@tournament_preferences_registry.register
+class AllowSearchEngineIndexing(ChoicePreference):
+    help_text = _("Set whether or not search engines can index the tab."
+                  "This is done by adding a <meta name=\"robots\" content=\"noindex\">"
+                  "tag to appropriate pages. Please note that while most search"
+                  "engines (e.g. Google) do respect this, not all search engines"
+                  "will. While this may be useful in preserving participants"
+                  "privacy, you may wish to also consider other techniques"
+                  "(such as disabling public pages or password-protecting your)"
+                  "web server.")
+    verbose_name = _("Search engine indexing configuration")
```

Comment:
This feels too technical... how about something like:

Help text:
> Adds a flag signaling search engines to not include tournament pages in search results. Note that it may get ignored, and the other available settings will provide greater privacy.

Verbose name:
> Include site in search engines